From b32bafc6130434972ffcd245e8f7aabca767e704 Mon Sep 17 00:00:00 2001
From: Zhi Jin <zhi.jin@intel.com>
Date: Fri, 6 Sep 2019 19:21:33 +0800
Subject: [PATCH 2/2] Disable usb-otg-switch on AaaS

Otherwise AaaS fails to boot on some USB port.

Change-Id: I942a3208a4ced4210b3b800e2c3d1f11566b8eb7
Tracked-On:
Signed-off-by: Zhi Jin <zhi.jin@intel.com>
---
 aaas/mixins.spec | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/aaas/mixins.spec b/aaas/mixins.spec
index cca96d4..8e904e8 100755
--- a/aaas/mixins.spec
+++ b/aaas/mixins.spec
@@ -59,7 +59,7 @@ firststage-mount: true
 cpuset: autocores
 usb-init: true
 usb-audio-init: true
-usb-otg-switch: true
+usb-otg-switch: false
 vndk: true
 public-libraries: true
 device-specific: aaas
-- 
2.7.4

